frontend/src/lib/components/PropertyKeyInfo.tsx

```diff
@@ -645,7 +645,7 @@ export function PropertyKeyInfo({
     value = `${value}` // convert to string
 
     const data = getKeyMapping(value, type)
-    const baseValue = (!data ? value : data.label).trim()
+    const baseValue = (!data ? value : data.label)?.trim() ?? ''
```

Comment:
nit but I think things are more readable if it doesn't have to be negated

```suggestion
    const baseValue = (data ? data.label : value)?.trim() ?? ''
```

Reply:
good call 👍